src/components/login-dropdown.tsx

```diff
@@ -17,15 +17,99 @@ interface ChaiseLoginDropdownProps {
   menu: MenuOption[],
   openProfileCb?: MouseEventHandler,
   parentDropdown: any // TODO: useRef wrapper type
+  alignRight: boolean
```

Comment:
You don't need to worry about the existing props, but please add a comment explaining what this prop is

```
/**
 * Explain its purpose and how its used
 */ 
alignRight: boolean
```